uber/custom_tags.py

```diff
@@ -133,6 +145,7 @@ def email_only(email):
 
 @tag
 class maybe_anchor(template.Node):
+
```

Comment:
Running autopep8 seems to have added a whole lot of extra lines where the code didn't have it before. :( I don't know if you can undo that easily or not -- if you can, please do. This code was passing our pep8 checks before, so these extra lines aren't needed.